chord_metadata_service/chord/ingest.py

```diff
@@ -247,4 +269,5 @@ def ingest_phenopacket(phenopacket_data, table_id) -> pm.Phenopacket:
 DATA_TYPE_INGEST_FUNCTION_MAP = {
     DATA_TYPE_EXPERIMENT: ingest_experiment,
     DATA_TYPE_PHENOPACKET: ingest_phenopacket,
+    DATA_TYPE_FHIR: ingest_fhir
```

Comment:
@davidlougheed To what type do I set here 'ingest_fhir' view then? (if I remove fhir data type)

Reply:
I'll take a look - I think i'll need to tweak it a bit to check the workflow ID. I'll try to do that before 2pm or so

Reply:
Ok thanks!